php_laravel/config/omise.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+use Illuminate\Support\Facades\Facade;
+use Illuminate\Support\ServiceProvider;
+
+return [
+
+    
+    'omise_public_key' => env('OMISE_PUBLIC_KEY', '')
```

Comment:
We need to add secret key as well.

Reply:
sure, will add.

Reply:
@aashishgurung 
Done
https://github.com/omise/examples/pull/29/commits/64d875dae2316e04d267dd2750d5c4c4f75ceae5